Update peerDeps to support Svelte 5 (@uploadthing/svelte@7.0.3)

### Describe the feature you'd like to request

After upgrading to Svelte 5 and running npm install I get:

```
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: navar@0.0.1
npm ERR! Found: svelte@5.0.5
npm ERR! node_modules/svelte
npm ERR!   dev svelte@"^5.0.5" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer svelte@"^4.0.0" from @uploadthing/svelte@7.0.3
npm ERR! node_modules/@uploadthing/svelte
npm ERR!   @uploadthing/svelte@"^7.0.3" from the root project
npm ERR!
```
Although I can do `npm install --legacy-peer-deps`

Just need to update the peerDependencies as I can see that ^5.0.0-next.0 was already supported.

### Describe the solution you'd like to see

Upgrade peer deps

### Additional information

_No response_

### 👨‍👧‍👦 Contributing

- [ ] 🙋‍♂️ Yes, I'd be down to file a PR implementing this feature!

--- a/.changeset/flat-seals-change.md
+++ b/.changeset/flat-seals-change.md
@@ -1,0 +1,5 @@
+---
+"@uploadthing/svelte": patch
+---
+
+chore: update svelte peer dep range

--- a/packages/svelte/package.json
+++ b/packages/svelte/package.json
@@ -29,7 +29,7 @@
     "!dist/**/*.spec.*"
   ],
   "peerDependencies": {
-    "svelte": "^4.0.0 || ^5.0.0-next.0",
+    "svelte": "^4.0.0 || ^5.0.0",
     "uploadthing": "^7.0.0"
   },
   "dependencies": {

